src/PointwiseFunctions/AnalyticData/NewtonianEuler/KhInstability.hpp

```diff
@@ -0,0 +1,245 @@
+ * initialized to a small perturbation
+ *
+ * \f{align*}
+ * v_y(x, y) = A\sin(4\pi x)
+ * \left[\exp\left(-\dfrac{(y - y_\text{top})^2}{2\sigma^2}\right) +
+ * \exp\left(-\dfrac{(y - y_\text{bot})^2}{2\sigma^2}\right)\right],
+ * \f}
+ *
+ * where \f$A\f$ is the amplitude, \f$\sigma\f$ is a damping factor, and
+ * \f$y_\text{top} = y_\text{mid} + b/2\f$ and
+ * \f$y_\text{bot} = y_\text{mid} - b/2\f$ are the vertical coordinates
+ * of the top and bottom boundaries of the strip, respectively.
+ *
+ * \note This class can be used to initialize 2D and 3D data. In 3D, the strip
+ * is aligned to the \f$x-\f$axis, and the vertical direction is taken to be
+ * the \f$z-\f$axis.
+ */
+template <size_t Dim>
+class KhInstability : public MarkAsAnalyticData {
+ public:
+  using equation_of_state_type = EquationsOfState::IdealFluid<false>;
+  using source_term_type = Sources::NoSource;
+
+  /// The adiabatic index of the fluid.
+  struct AdiabaticIndex {
+    using type = double;
+    static constexpr OptionString help = {"The adiabatic index of the fluid."};
+  };
+
+  /// The vertical coordinate of the horizontal bimedian of the strip.
+  struct StripBimedian {
+    using type = double;
+    static constexpr OptionString help = {"The height of the strip center."};
+  };
+
+  /// The thickness of the strip.
+  struct StripThickness {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static constexpr OptionString help = {
+        "The thickness of the horizontal strip."};
+  };
+
+  /// The mass density in the strip
+  struct StripDensity {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static constexpr OptionString help = {
+        "The mass density below the interface"};
+  };
+
+  /// The velocity along \f$x\f$ in the strip
+  struct StripVelocity {
+    using type = double;
+    static constexpr OptionString help = {
+        "The velocity along x below the interface."};
+  };
+
+  /// The mass density outside of the strip
+  struct BackgroundDensity {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static constexpr OptionString help = {
+        "The mass density above the interface."};
+  };
+
+  /// The velocity along \f$x\f$ outside of the strip
+  struct BackgroundVelocity {
+    using type = double;
+    static constexpr OptionString help = {
+        "The velocity along x above the interface."};
+  };
+
+  /// The initial (constant) pressure of the fluid
+  struct Pressure {
+    using type = double;
+    static type lower_bound() noexcept { return 0.0; }
+    static constexpr OptionString help = {"The initial (constant) pressure."};
+  };
+
+  /// The amplitude of the perturbation
+  struct PerturbAmplitude {
+    using type = double;
+    static constexpr OptionString help = {"The amplitude of the perturbation."};
+  };
+
+  /// The damping factor of the perturbation
+  struct DampingFactor {
```

Comment:
I would suggest something more descriptive, like `PerturbWidth`. The current `DampingFactor` doesn't tell me what is being damped and in what way...